Libraries/CMSIS/Device/Maxim/MAX32690/Include/max32690.svd

```diff
@@ -4805,7 +4805,7 @@ least 1 entry in RX FIFO during this time, this bit is clear by write 1</descrip
      <name>CLKDIV</name>
      <description>Flash Clock Divide. The clock (PLL0) is divided by this value to generate a 1 MHz clock for Flash controller.</description>
      <addressOffset>0x04</addressOffset>
-     <resetValue>0x00000064</resetValue>
+     <resetValue>0x00000078</resetValue>
```

Comment:
Could you also update the reset value in [Libraries/PeriphDrivers/Source/FLC/flc_reva_me30.svd](https://github.com/perkinsmg/msdk/blob/fix/flc-clkdiv-reset-value/Libraries/PeriphDrivers/Source/FLC/flc_revb_me30.svd#L37C9-L37C44)?